kbase-extension/static/kbase/js/kbaseNarrative.js

```diff
@@ -166,29 +165,24 @@ define([
     };
 
     Narrative.prototype.getNarrativeRef = function () {
-        return Promise.try(() => {
-            if (this.workspaceRef) {
+        if (this.workspaceRef) {
```

Comment:
Did a little unraveling around the Promise structure. Code's effectively unchanged.